src/ethereum/exceptions.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
Perhaps we should rename this file to `error.py` for consistency with the analogous file in `vm/`?